config/routes.rb

```diff
@@ -52,6 +52,7 @@
     resources :listings, only: %i[index edit update destroy], controller: "classified_listings"
     resources :pages, only: %i[index new create edit update destroy]
     resources :mods, only: %i[index update]
+    resources :moderator_actions, only: %i[index]
```

Comment:
can't find this in the internal menu, should it be hidden?

Reply:
Added! Was waiting until I rebased to do this :)